### PR TITLE
Add: Added JS sample code for mouse actions in support packages

### PR DESCRIPTION
**Pull request by Harsha509**
Signed-off-by: Sri Harsha <sri_harsha509@hotmail.com>

**Thanks for contributing to the Selenium site and documentation!**
**A PR well described will help maintainers to quickly review and merge it**

Before submitting your PR, please check our [contributing](https://selenium.dev/documentation/en/contributing/) guidelines.
Avoid large PRs, help reviewers by making them as simple and short as possible.

### Types of changes

- [ ] Change to the site (I am attaching a screenshot showing the before and after)
- [x] Code example added (and I also added the example to all translated languages)
- [ ] Improved translation
- [x] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [x] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [x] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

### CODE REVIEW

File: docs_source_files/content/support_packages/mouse_and_keyboard_actions_in_detail.de.md [hunk 1]
```diff
@@ -101,7 +101,7 @@ actions.perform()
 # We don't have a Ruby code sample yet -  Help us out and raise a PR
   {{< / code-panel >}}
   {{< code-panel language="javascript" >}}
-// We don't have a JavaScript code sample yet -  Help us out and raise a PR
+ await actions.move({origin: e}).perform();
```
Comment (manoj9788) [inline docs_source_files/content/support_packages/mouse_and_keyboard_actions_in_detail.de.md]: Let's try to be consistent with examples of other languages with a complete syntax. Would the below help?
await driver.actions()
          .move({origin: e})
          .perform();

Comment (Harsha509) [reply docs_source_files/content/support_packages/mouse_and_keyboard_actions_in_detail.de.md]: HI @manoj9788 ,

Yes, as the above code has the driver declaration 'const actions = driver.actions();'. i have updated code as
 await actions
 .move({origin: e})
 .perform();

**Comment by Harsha509** [comment]
> Can you also please add code for the other two sections of this page?
> Enter a sequence of characters in e:
> and
> Delete the text that is in e (if there is any):

Done.

**Comment by manoj9788** [comment]
Thank you @Harsha509

**Comment by Harsha509** [comment]
Thanks for reviewing @manoj9788 .

**Pull request merged**